docs/static_landing_page/js/landing_page.js

```diff
@@ -148,7 +148,7 @@ $(document).ready(function () {
 
 async function setLinks() {
   // update this version with every release
-  const latestVersion = 'v0.9'; // Do not include the patch
+  const latestVersion = 'v0.10.0'; // Do not include the patch
```

Comment:
I'm not sure why the comment about patch is there but that's the last of the three numbers. So 0.10 only? @riedgar-ms any idea why?

Reply:
I'm not sure about that.... possibly @adrinjalali ? But if you look at our docs, the URIs are like:
https://fairlearn.org/v0.9/api_reference/index.html
I suspect that the reasoning is that 'patch' changes should only be bug/typo fixes, and so not really another 'version' of the docs.

Reply:
I removed the comment and changed the URL to 0.10

Reply:
@romanlutz  and @riedgar-ms  can you confirm the change is present? If so, I will resolve this comment